example/jitter/unpack_upx.py

```diff
@@ -59,7 +59,8 @@ def kernel32_GetProcAddress(jitter):
 assert(len(leaves) == 1)
 l = leaves.pop()
 logging.info(l)
-end_label = l.label.offset
+
+end_label = mdis.symbol_pool.loc_key_to_offset(l)
```

Comment:
If this script is used as a base for future ones, `label*` name may be misleading

Reply:
Correct!
done